internal/lookoutv2/repository/common_test.go

```diff
@@ -0,0 +1,94 @@
+package repository
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/G-Research/armada/internal/common/util"
+)
+
```

Comment:
There's fewer tests here than I would expect given the complexity of the code in `common.go`